NYTPhotoViewer/include/NYTPhotosViewController.h

```diff
@@ -41,6 +41,12 @@ extern NSString * const NYTPhotosViewControllerDidDismissNotification;
 
 @interface NYTPhotosViewController : UIViewController
 
+typedef NS_ENUM(NSUInteger, NYTPhotoViewerStatusBarMode) {
+    NYTPhotoViewerStatusBarModeDynamic, /* hide statusbar when view.safeArea.top == 0 */
```

Comment:
A doc comment instead of a regular comment would make this show up in the inline doc viewer. Probably best to add comments for all the cases.